### PR TITLE
WOE 7 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/blossoming_tortoise.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Blossoming Tortoise
+ManaCost:2 G G
+Types:Creature Turtle
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, mill three cards, then return a land card from your graveyard to the battlefield tapped.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigMill | Secondary$ True | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, mill three cards, then return a land card from your graveyard to the battlefield tapped.
+SVar:TrigMill:DB$ Mill | Defined$ You | NumCards$ 3 | SubAbility$ DBReturn
+SVar:DBReturn:DB$ ChangeZone | ChangeType$ Land.YouOwn | Origin$ Graveyard | Destination$ Battlefield | Hidden$ True | ChangeNum$ 1 | Tapped$ True
+S:Mode$ ReduceCost | ValidCard$ Land.YouCtrl | Type$ Ability | Amount$ 1 | AffectedZone$ Battlefield | Description$ Activated abilities of lands you control cost {1} less to activate.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/blossoming_tortoise.txt:9]: Probably Type Activated?

File: forge-gui/res/cardsfolder/upcoming/callous_sell_sword_burn_together.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+DeckHas:Ability$Counters|Sacrifice
+AlternateMode:Adventure
+Oracle:Callous Sell-Sword enters the battlefield with a +1/+1 counter on it for each creature that died under your control this turn.
+
+ALTERNATE
+
+Name:Burn Together
+ManaCost:R
+Types:Sorcery Adventure
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | RememberObjects$ ThisTargetedCard | TgtPrompt$ Select target creature you control | SubAbility$ DBDamage | SpellDescription$ Target creature you control deals damage equal to its power to any other target. Then sacrifice it.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/callous_sell_sword_burn_together.txt:17]: @Agetian AILogic for Bite effects?

File: forge-gui/res/cardsfolder/upcoming/blossoming_tortoise.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/blossoming_tortoise.txt:9]: doesn't exist

File: forge-gui/res/cardsfolder/upcoming/beluna_grandsquall.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+AlternateMode:Adventure
+Oracle:Trample\nPermanent spells you cast that have an Adventure cost {1} less to cast.
+
+ALTERNATE
+
+Name:Seek Thrills
+ManaCost:2 G U R
+Types:Instant Adventure
+A:SP$ Mill |  NumCards$ 7 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill seven cards. Then put all cards that have an Adventure from among the milled cards into your hand.
+SVar:DBChangeZone:DB$ ChangeZoneAll | ChangeType$ Creature.IsRemembered+AdventureCard | Origin$ Graveyard | Destination$ Hand | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/beluna_grandsquall.txt:18]: wrong type

File: forge-gui/res/cardsfolder/upcoming/blossoming_tortoise.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Blossoming Tortoise
+ManaCost:2 G G
+Types:Creature Turtle
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, mill three cards, then return a land card from your graveyard to the battlefield tapped.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigMill | Secondary$ True | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, mill three cards, then return a land card from your graveyard to the battlefield tapped.
+SVar:TrigMill:DB$ Mill | Defined$ You | NumCards$ 3 | SubAbility$ DBReturn
+SVar:DBReturn:DB$ ChangeZone | ChangeType$ Land.YouOwn | Origin$ Graveyard | Destination$ Battlefield | Hidden$ True | ChangeNum$ 1 | Tapped$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/blossoming_tortoise.txt:8]: does it need Mandatory?

File: forge-gui/res/cardsfolder/upcoming/beanstalk_wurm_plant_beans.txt [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+K:Reach
+AlternateMode:Adventure
+Oracle:Reach
+
+ALTERNATE
+
+Name:Plant Beans
+ManaCost:1 G
+Types:Sorcery Adventure
+A:SP$ Effect | Name$ Explore Effect | StaticAbilities$ Exploration | AILogic$ Always | SpellDescription$ You may play an additional land this turn. (Then exile this card. You may caste the creature later from exile.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/beanstalk_wurm_plant_beans.txt:14]: "caste"

File: forge-gui/res/cardsfolder/upcoming/beluna_grandsquall.txt [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+AlternateMode:Adventure
+Oracle:Trample\nPermanent spells you cast that have an Adventure cost {1} less to cast.
+
+ALTERNATE
+
+Name:Seek Thrills
+ManaCost:2 G U R
+Types:Instant Adventure
+A:SP$ Mill |  NumCards$ 7 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill seven cards. Then put all cards that have an Adventure from among the milled cards into your hand.
+SVar:DBChangeZone:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered+AdventureCard | Origin$ Graveyard | Destination$ Hand | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/beluna_grandsquall.txt:18]: Add Exile as Origin

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/callous_sell_sword_burn_together.txt (hunk 2 above)
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/callous_sell_sword_burn_together.txt:17]: @Simisays  `AILogic$ PowerDmg` see https://github.com/Card-Forge/forge/blob/master/forge-gui/res/cardsfolder/b/bite_down.txt

**Review by tool4ever — APPROVED** [review]

**Pull request merged**